### DISCUSSION TITLE
Boolean field has wrong height

### DISCUSSION THREAD

**Discussion by williamcruzme** [laravel/nova-issues#3938] [Q&A]
Laravel Version: 8.83.7
Nova Version: 15.7.0
PHP Version: 8.0
Database Driver & Version: MySQL 8.0
Operating System and Version: macOS 12.3.1
Browser type and version: Chrome Version 100.0.4896.75 arm64

### Description:

<img width="590" alt="image" src="https://user-images.githubusercontent.com/22250407/162671909-b0d713f2-a0ce-4f3b-a63c-a1ecc709fd13.png">

**Comment by davidhemphill** [comment]
What are you referring to specifically? It's not clear to me from the image what you're referring to.

**Reply by williamcruzme** [reply]
<img width="923" alt="image" src="https://user-images.githubusercontent.com/22250407/162842070-1b9cc74d-786e-43f7-8051-e412b790e17e.png">

If you check the padding top and padding bottom for the Boolean field, it's not perfect, like the rest of the fields.

That issue is more visible if the Boolean field is the last field of the panel.
